documentation/modules/exploit/windows/misc/cisco_webex_ext.md

```diff
@@ -0,0 +1,52 @@
+Cisco WebEx is a suite of applications for online meeting organization and video conferencing.
+Included in this suite are extensions for popular web browsers which ease use and provide supplemental
+features.
+
+Version 1.0.1 of the WebEx extension for Google Chrome contains a vulnerability which allows an
+attacker to execute arbitrary commands on a target, which can lead to arbitrary remote code execution.
+
+
+## Vulnerable Application
+
+Cisco WebEx Chrome Extension 1.0.1 is known to be affected.
+
+## Verification Steps
+
+**Using cisco_webex_ext**
```

Comment:
I think this line can be removed, since its the docs for this module, should be self evident.